pom.xml

```diff
@@ -276,6 +281,7 @@
                                 <infinispan.image>quay.io/infinispan/server:14.0</infinispan.image>
                                 <infinispan.expected-log>Infinispan Server.*started in</infinispan.expected-log>
                                 <spring.cloud.server.image>quay.io/quarkusqeteam/spring-cloud-config-server:3.0</spring.cloud.server.image>
+                                <keycloak.image>quay.io/keycloak/keycloak:22.0.5</keycloak.image>
```

Comment:
I'm afraid you should wait for https://github.com/quarkus-qe/quarkus-test-suite/pull/1581 to get in and then rebase, not sure whether 22.0.5 is going to be needed.